content/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph.md

```diff
@@ -73,7 +73,7 @@ The recommended formats explicitly define which versions are used for all direct
 | --- | --- | --- | ---|
 | Composer             | PHP           | `composer.lock` | `composer.json`, `composer.lock` |
 | `dotnet` CLI | .NET languages (C#, C++, F#, VB)  |   `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj` |  `.csproj`, `.vbproj`, `.nuspec`, `.vcxproj`, `.fsproj`, `packages.config` |{% ifversion fpt or ghes > 3.1 %}
-| Go modules | Go | `go.mod` | `go.mod` |{% endif %}
+| Go modules | Go | `go.mod`, `go.sum` | `go.mod`,`go.sum` |{% endif %}
```

Comment:
According to the (strange) definition above, only go.sum is "recommended".

(The definition is strange because what it's really talking about is the difference between manifests that enumerate transitive dependencies versus those that enumerate only direct edges. A user--of Go at least--doesn't choose which manifest type to use, so "recommended" doesn't really make sense.)

Reply:
Have you uploaded your change? I don't see it.

Reply:
updated